py/torch_tensorrt/dynamo/fx_ts_compat/lower.py

```diff
@@ -67,7 +71,11 @@ def compile(
         timing_cache_prefix: Timing cache file name for timing cache used by fx2trt.
         save_timing_cache: Update timing cache with current timing cache data if set to True.
         cuda_graph_batch_size: Cuda graph batch size, default to be -1.
+        explicit_batch_dimension: Whether to specify an explicit batch dimension to TRT
         use_experimental_fx_rt: Uses the next generation TRTModule which supports both Python and TorchScript based execution (including in C++).
+        max_aux_streams: max number of aux stream to use
+        version_compatible: enable version compatible feature
```

Comment:
Can you let me know what version_compatible is and what it does?

Reply:
This was introduced in #1935, but I am just ensuring users can call it from the frontend API. In the TRT documentation, [version_compatible](https://docs.nvidia.com/deeplearning/tensorrt/api/python_api/infer/Core/BuilderConfig.html?highlight=version_compatible) controls whether the engine plan file is forward-compatible.